src/js/components/epics/progress.tsx

```diff
@@ -25,6 +27,16 @@ const EpicProgress = ({ range }: Props) => {
         thickness="small"
         color={value === 100 ? 'success' : undefined}
       />
+      <TourPopover
+        align="right"
+        heading={i18n.t('Task progress bar')}
+        body={
+          <Trans i18nKey="tourTaskProgress">
+            A quick reference to show what percentage of an Epic is complete
+            based on the number of Tasks that are complete.
+          </Trans>
+        }
+      />
     </div>
```

Comment:
I don't know the best place to leave this @dvdherron, but I see tis progress bar description is also showing up where the progress path description should be. 
![task-progress-path](https://user-images.githubusercontent.com/1581694/119443322-0dd1f000-bcef-11eb-9337-3366343b1f86.jpg)

Reply:
Good catch. Both elements had the same translation key. Fixed. 
![image](https://user-images.githubusercontent.com/41588129/119477574-5c16db80-bd4f-11eb-81e4-4db80eccac7e.png)